server/blogupdate.js

```diff
@@ -0,0 +1,137 @@
+/* Database configuration */
+const firebaseConfig = {
```

Comment:
have one single file that has these configurations and call it wherever it is needed